Plan/bukkit/src/main/java/com/djrapitops/plan/gathering/listeners/bukkit/PlayerOnlineListener.java

```diff
@@ -85,12 +85,14 @@ public void onPlayerLogin(PlayerLoginEvent event) {
 
             String address = event.getHostname();
             if (!address.isEmpty()) {
-                address = address.substring(0, address.lastIndexOf(':'));
-                if (address.contains("\u0000")) {
-                    address = address.substring(0, address.indexOf('\u0000'));
-                }
-                if (address.contains("fml")) {
-                    address = address.substring(0, address.lastIndexOf("fml"));
+                if (address.contains(":")) {
```

Comment:
This if-block should terminate before the next if block instead of wrapping everything (for clarity) even though the null character and fml blocks come after it

Reply:
> This if-block should terminate before the next if block instead of wrapping everything (for clarity) even though the null character and fml blocks come after it

Thanks to point out.